server/channels/api4/system.go

```diff
@@ -631,15 +631,19 @@ func pushNotificationAck(c *Context, w http.ResponseWriter, r *http.Request) {
 	err := c.App.SendAckToPushProxy(&ack)
 	if ack.IsIdLoaded {
 		if err != nil {
-			// Log the error only, then continue to fetch notification message
```

Comment:
Why are we removing this comment?

Reply:
Didn't seem super necessary, we can tell the app isn't `return`ing